LiteFarmOrg/LiteFarm
# Lf 2888 allow genus and species to be specified in the add new crop workflow

Issue: [LF-2888](https://lite-farm.atlassian.net/browse/LF-2888)
Parent issue: [LF-2072](https://lite-farm.atlassian.net/browse/LF-2072)

**Description**
Adds inputs for genus and species on new crop creation. Adds translations placeholders for Genus and Species. 

**Test**
1. Navigate to crop catalogue
2. Create custom crop
3. Genus and species fields should be present, validation creates an error message for >200 chars
4. Genus and species value should be persisted in database as Genus: first letter capitalized, species: lowercase.
